qiskit/schemas/qobj_schema.json

```diff
@@ -251,69 +251,33 @@
         },
         "openqasm_instructions": {
             "oneOf": [
-                {
+                { 
                     "properties": {
-                        "name": {
-                            "enum": [
-                                "snapshot",
-                                "wait",
-                                "noise",
-                                "save",
-                                "load"
-                            ]
-                        }
-                    },
-                    "title": "Simulator instructions"
+                        "name": {"not": {
+                            "enum": ["snapshot", "reset", "barrier", "bfunc", "copy", "measure"]
```

Comment:
I don't think that Qobj should be aware of the extensions. That does not scale very well since adding new extensions would mean modifying the schema. Don't you think that extensions should be independent of the base Qobj schema?

Reply:
It is not aware. It is specifying a set of basic, core instructions. And it says if an instruction is not part of those, then leave it to the backend to validate.
I agree that `snapshot` shouldn't be here. That's very simulator specific. But `reset`, `barrier`, `bfunc`, `copy`, `measure` are basic qasm instructions.
I recently argued for adding all of `qelib1` here as well.

Reply:
I guess my philosophy was to leave snapshot in becuase it doesn't impact anything if ran on a device (device will just ignore it)